lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -2415,8 +2415,16 @@ checkIndividualConformance(NormalProtocolConformance *conformance) {
       ComplainLoc, diag::unchecked_conformance_not_special, ProtoType);
   }
 
+  bool allowImpliedConditionalConformance = false;
+  if (Proto->isSpecificProtocol(KnownProtocolKind::Sendable)) {
+    if (Context.LangOpts.StrictConcurrencyLevel != StrictConcurrency::Complete)
```

Comment:
Should this be checking for Swift 6 instead, so we don't change Swift 5 code that enabled strict checking?